models.py

```diff
@@ -83,28 +110,15 @@ class Comments(db.Model):
     user_id = db.Column(db.Integer, db.ForeignKey('users.user_id'))
     commenter_id = db.Column(db.Integer, db.ForeignKey('users.user_id'))
     comment = db.Column(db.String(100))
-    rating = db.Column(db.Float)
-    datetime = db.Column(db.DateTime)
+    rating = db.Column(db.Integer)
+    date_time = db.Column(db.DateTime)
```

Comment:
when did you leave the comment to the user. You might want to add multiple comments over time. This will just give an idea when you commented a particular provider.